lib/index.js

```diff
@@ -114,27 +115,21 @@ Nanigans.prototype.track = function(track) {
       params.products.value = data.value;
       params.products.sku = data.sku;
       params.products = qs.stringify(params.products);
-      this.load('purchase', params);
+      params.userId ? this.load('purchase', params) : this.load('purchase_no_user_id', params);
       break;
     case 'user':
       switch (event.name) {
-      case 'product':
```

Comment:
@hankim813 - I was a bit lazy on the query writing. I did, however, look at every record, and there aren't any with a Nanigans event name of `product`.